docs/api_documentation.md

```diff
@@ -651,6 +644,15 @@ e.g.:
 curl -i -H "Accept: application/json" -H "Content-type: application/json" -H "Authorization: Bearer $YOUR_ACCESS_TOKEN" -X DELETE http://localhost:8080/osiam/Users/$ID
 ```
 
+### Get the Authenticated User
```

Comment:
Sure. But I think we should fix any error in the documentation to have something to start with.